core.md

```diff
@@ -371,23 +416,44 @@ placed into a weakmap.
 
 ## Prototypes
 
-FIXME
+All typed objects have an accompanying `prototype`. The `[[Prototype]]` of new instances of a type is set to that `prototype`. For struct types, the `prototype`'s own `[[Prototype]]` is immutably set to `StructType.prototype`. The `[[Prototype]]` of arrays of a struct type `Point`, instantiated using `new Point.array()`, is set to `Point.array.prototype`. That object's own `[[Prototype]]` is set to `StructType.array.prototype`.
+
+In code:
+
+```js
+const Point({x: float64, y: float64});
+const Line({start: Point, end: Point});
+let point = new Point();
+let points1 = new Point.array(2);
+let points2 = new Point.array(5);
+let line = new Line();
+// These all yield `true`:
+point.__proto__ === Point.prototype;
+points1.__proto__ === Point.array.prototype;
+points2.__proto__ === points1.__proto__;
+Point.prototype.__proto__ === StructType.prototype;
+Point.array.prototype.__proto__ === StructType.array.prototype;
+line.__proto__ === Line.prototype;
```

Comment:
I added this, yes. The reason I didn't before was that I was on the fence about it. I've come down on the "yes" side of the fence now, though. The only question is whether there should be a common base for value and struct types and their arrays, but that's for a later change.